pitch.md

```diff
@@ -13,14 +14,16 @@ good way to meet their needs.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
-
+A DSL is appropriate for my users because in any given project there may be different keywords, group sizes, leadership positions, etc. that are unique to this project. A language allows the teacher to define what constraints matter and which constraints dont and provide her the flexibility to even input her own personal preferences about who would and would not work together etc. It addresses the need by allowing the teacher to simply input text that makes up the descriptions of each student and have that function as a program to return the groups that she needs.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+I came up with this idea in discussion with Prof. Ben and since then I have gotten increasingly excited about it. I think that constraint solving is very interesting, but more importantly I think that the language design aspects of this project are intriguing. Additionally, from a user standpoint, I have been in many bad groups and many good ones and can testify first hand what a large difference a good group can make to my learning and the quality of my final product.
```

Comment:
Everybody would love a tool like this. It will be interesting to see how much of the group dynamics problem can be solved through just constraints.